src/db/db_bench.cc

```diff
@@ -947,7 +948,10 @@ class Benchmark {
       } else if (name == Slice("printdb")) {
     	fresh_db = false;
     	method = &Benchmark::PrintDB;
-      } else {
+      } else if (name == Slice("filter")) {
+        PrintStats("leveldb.filter");
```

Comment:
This is great, thanks! As before, could we have the output printed at the end of the benchmark run?

Reply:
Currently, I was printing it by passing it to `--benchmarks=fillrandom,readrandom,filter`.
Do you want it to be printed by default without passing it to as a flag?

Reply:
Ah, I see. No, then some documentation or some examples showing how it would be used should be enough.

Reply:
Will it be fine to update the sample usage in  the [README](https://github.com/utsaslab/pebblesdb#running-microbenchmark)?

Reply:
That should be good.

Reply:
Please have a look at the updated readme and let me know if it's fine.